include/z64save.h

```diff
@@ -729,7 +735,10 @@ typedef enum {
 #define WEEKEVENTREG_22_01 PACK_WEEKEVENTREG_FLAG(22, 0x01)
 
 #define WEEKEVENTREG_22_02 PACK_WEEKEVENTREG_FLAG(22, 0x02)
-#define WEEKEVENTREG_22_04 PACK_WEEKEVENTREG_FLAG(22, 0x04)
+
+// Goron Elder's son has been calmed.
+#define WEEKEVENTREG_CALMED_GORON_ELDERS_SON PACK_WEEKEVENTREG_FLAG(22, 0x04)
```

Comment:
```suggestion
// Goron Elder's son has been calmed from Goron's Lullaby
#define WEEKEVENTREG_CALMED_GORON_ELDERS_SON PACK_WEEKEVENTREG_FLAG(22, 0x04)
```
Maybe you should mention Lullaby in the comment somehow?